protected/views/site/guideepigenomic.php

```diff
@@ -3,18 +3,6 @@
 
```

Comment:
all changes in the following files are analogous to the ones in guide.php, only outstanding differences will be noted